google/cloud/bigquery/enums.py

```diff
@@ -49,6 +49,24 @@ class Compression(object):
     """Specifies no compression."""
 
 
+class DecimalTargetType:
```

Comment:
I notice we are missing docs for a bunch of enums. I filed https://github.com/googleapis/python-bigquery/issues/744 to switch the docs over to module-based docs for this module.